test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
As the content of the file says, this file should be ignored by git. You might want to have a look back at the .gitignore file and check whether you are making an error there 😊. But anyhow, this file should not be present on GitHub.